app/tasks.py

```
@@ -401,401 +401,405 @@ def sync_all_users():
+            # AEMO mode: No WebSocket, use AEMO API for forecast
+            # Current price will be derived from the first forecast interval
+            logger.info(f"📊 AEMO mode - fetching forecast from AEMO for region {user.flow_power_state}")
+        elif websocket_data:
+            # WebSocket data received within 60s - use it directly as primary source
+            current_actual_interval = websocket_data
+            general_price = current_actual_interval.get('general', {}).get('perKwh') if current_actual_interval.get('general') else None
+            feedin_price = current_actual_interval.get('feedIn', {}).get('perKwh') if current_actual_interval.get('feedIn') else None
+            logger.info(f"✅ Using WebSocket price for current interval: general={general_price}¢/kWh, feedIn={feedin_price}¢/kWh")
+        else:
+            # WebSocket timeout - fallback to REST API for current price
+            logger.info(f"⏰ Fetching current price from REST API")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)